hooks/schema-generation.js

```diff
@@ -0,0 +1,41 @@
+const {TypeScriptGenerator, FormatHelpers} = require('@asyncapi/generator-model-sdk');
+const fs = require('fs');
+const Path = require('path');
+
+const preset = {
+  interface: {
+    async property({ renderer, propertyName, property }) {
+      const name = propertyName;
```

Comment:
What's the point of this variable? It's because it's shorter? You can also alias it in the line above, I think it is something like this:

```
    async property({ renderer, propertyName: name, property }) {
```

Reply:
True, I will change it 😄